StorageUtility/Scsi.h

```diff
@@ -29,19 +29,83 @@ namespace vtStor
         {
             extern const U32 VTSTOR_API SCSI_SECTOR_SIZE_IN_BYTES;
 
-            struct sCdbFields
+#pragma pack(push, cdb, 1)
+
+            struct InputFields_Command6
+            {
+                U8  OpCode;
+                U8  Service;
+                U32 Lba;
+                U16 Length;
+                U8  Control;
+            };
+
+            struct InputFields_Command10
+            {
+                U8  OpCode;
+                U8  Service;
+                U32 Lba;
+                U16 Length;
+                U8  Control;
+                U8  MiscCdb;
+            };
+
+            struct InputFields_Command12
+            {
+                U8  OpCode;
+                U8  Service;
+                U32 Lba;
+                U32 Length;
+                U8  Control;
+                U8  MiscCdb;
+            };
+
+            struct InputFields_Command16
             {
                 U8  OpCode;
                 U8  Service;
                 U64 Lba;
-                U32 TransferLen;
-                U8  ParameterLen;
-                U32 AllocationLen;
-                U8  Group;
+                U32 Length;
+                U8  Control;
+                U8  MiscCdb;
+            };
+
+            struct InputFields_AtaPassThrough16
```

Comment:
This structure will help to set up the parameters easier for ata pass through commands